io/shared/src/test/scala/fs2/io/file/FilesSuite.scala

```diff
@@ -545,8 +547,12 @@ class FilesSuite extends Fs2Suite with BaseFileSuite {
       Stream
         .resource(tempFile)
         .flatMap { p =>
-          Files[IO].list(p).void.recover { case ex: NotDirectoryException =>
-            assertEquals(ex.getMessage, p.toString)
+          Files[IO].list(p).void.recover {
+            case _: UncheckedIOException if isNative =>
+              assert(true)
```

Comment:
That's the first failing test. SN 0.5 throws `UncheckedIOException` instead of `NotDirectoryException`.

https://github.com/scala-native/scala-native/blob/v0.5.8/javalib/src/main/scala/java/nio/file/Files.scala

Reply:
https://github.com/scala-native/scala-native/issues/4432